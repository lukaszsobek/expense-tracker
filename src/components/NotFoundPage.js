import React from "react"
import { Link }  from "react-router-dom"

export const NotFoundPage = () => (
	<div>
        404 - go to the <Link to="/">main page</Link>
	</div>
)

export default NotFoundPage