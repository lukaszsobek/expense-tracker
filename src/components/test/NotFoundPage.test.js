import React from "react"
import { shallow } from "enzyme"

import { NotFoundPage } from "../NotFoundPage"

test("Renders NotFoundPage", () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot()
})
