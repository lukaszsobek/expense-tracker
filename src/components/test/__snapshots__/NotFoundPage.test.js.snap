// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders NotFoundPage 1`] = `
<div>
  404 - go to the 
  <Link
    replace={false}
    to="/"
  >
    main page
  </Link>
</div>
`;
